crates/runtime/src/component.rs

```diff
@@ -109,8 +135,7 @@ pub struct VMLowering {
 #[repr(C)]
 // Set an appropriate alignment for this structure where the most-aligned value
 // internally right now is a pointer.
-#[cfg_attr(target_pointer_width = "32", repr(align(4)))]
-#[cfg_attr(target_pointer_width = "64", repr(align(8)))]
+#[repr(align(16))]
```

Comment:
Why this change? Does the comment just above need updating to explain it?

Reply:
It turns out that without this the `VMContext` wasn't properly aligned so MIRI was indicating that under-aligned writes were happening (oh dear!). The comments and annotations here predate all that but yeah I should update things here.